src/middlewares/validator.js

```diff
@@ -1,13 +1,40 @@
 const createError = require('http-errors')
 const Ajv = require('ajv')
 const ajvKeywords = require('ajv-keywords')
+const ajvLocalize = require('ajv-i18n')
 const {deepEqual} = require('assert')
 
 let ajv
 let previousConstructorOptions
-const defaults = {v5: true, $data: true, allErrors: true}
+const defaults = {
+  v5: true,
+  coerceTypes: 'array', // important for query string params
+  allErrors: true,
+  useDefaults: true,
+  $data: true // required for ajv-keywords
+}
+
+module.exports = ({ inputSchema, outputSchema, ajvOptions, errorFormat = errors => errors }) => {
+  const acceptLanguage = require('accept-language')
```

Comment:
Instead re-doing this here, can we delegate somehow the work of processing the languages to the already available (and configurable) [HTTP Content Negotiation middleware](https://github.com/middyjs/middy/blob/master/docs/middlewares.md#httpcontentnegotiation)?

If we do that, we should just assume that the `event` object will contain a property called `preferredLanguage`.